packages/mobile/src/onboarding/contacts/ImportContactsScreen.tsx

```diff
@@ -1,6 +1,7 @@
 import Button, { BtnSizes, BtnTypes } from '@celo/react-components/components/Button.v2'
 import Switch from '@celo/react-components/components/Switch.v2'
 import TextButton from '@celo/react-components/components/TextButton.v2'
+import Checkmark from '@celo/react-components/icons/Checkmark'
 import colors from '@celo/react-components/styles/colors'
 import colorsV2 from '@celo/react-components/styles/colors.v2'
```

Comment:
nit: not related to this specific PR, but we should remove old `colors`

Reply:
Good catch, I'll remove the dep from this new file